app/templates/main.html

```diff
@@ -43,22 +35,288 @@ <h1>Able</h1>
           <label for="use-strict-bounds">Strict Bounds</label>
         </div>
       </div>
+        <!-- The div of the data we need to capture or work with somehow -->
       <div id="pac-container">
         <input id="pac-input" type="text"
             placeholder="Enter a location">
       </div>
+        <!-- This needs to be taken out and reworked -->
+      <div>
+        <form action="send_email_button">
+           <button type="submit">Send Email</button>
+         </form>
+         <form action="see_editor_picks">
+           <button type="submit">See the Editor Picks</button>
+         </form>
+      </div>
+        <!-- This needs to be taken out and reworked -->
     </div>
     <div id="map"></div>
     <div id="infowindow-content">
       <img src="" width="16" height="16" id="place-icon">
       <span id="place-name"  class="title"></span><br>
       <span id="place-address"></span>
     </div>
-    
-    <script type=text/javascript src="{{
-    url_for('static', filename='js/map.js') }}"></script>
+    <div id="sidenav">
```

Comment:
You can remove the sidebar code from the main.html since we have a consistent one in base.html